src/NUnitTestAdapter/NUnit.TestAdapter.csproj

```diff
@@ -7,6 +7,8 @@
     <TargetFrameworks>net35;netcoreapp1.0</TargetFrameworks>
     <CodeAnalysisRuleSet>..\..\Osiris.Extended.ruleset</CodeAnalysisRuleSet>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <SourceLinkOriginUrl>https://github.com/nunit/nunit3-vs-adapter</SourceLinkOriginUrl>
```

Comment:
Necessary because the `origin` remote URL is not necessarily `https://github.com/nunit/nunit3-vs-adapter`.